src/containers/ColorContainer.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ColorContainer from './ColorContainer';
+
+describe('ColorContainer component', () => {
+  it('renders ColorContainer', () => {
+    const wrapper = shallow(<ColorContainer />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Viewed
@@ -0,0 +1,12 @@
import React from 'react';
import { shallow } from 'enzyme';
import ColorContainer from './ColorContainer';

describe('Color Container', () => {

  it('renders color container component', () => {
    const wrapper = shallow(<ColorContainer />);
    expect(wrapper).toMatchSnapshot();
  });
 @meganamarshall
meganamarshall 8 minutes ago
there should be a state test here also, ie:

it('Changes Color on button click', () => {
const wrapper = mount(<ColorContainer colors={[
'red',
'green',
'black',
'yellow',
]} />);
wrapper.findWhere(el => el.key() === 'red').simulate('click');
expect(wrapper.state('backgroundColor')).toBe('red');
});